### DISCUSSION TITLE
Where is the CHANGELOG for 0.43.0?

### DISCUSSION THREAD

**Discussion by shantanugadgil** [vectordotdev/vector#21951] [Q&A] [answered]
It has been many (many) hours since the 0.43.0 release happened. The release notes' link is still broken. Any news about that?

**Comment by jszwedko** [comment]
These will be published today. Apologies for the delay, there were a few release hiccups yesterday and so publishing the notes hasn't yet been done.

**Comment by pront** [comment, marked as answer]
It is live now: https://vector.dev/releases/0.43.0/
Enjoy!

**Comment by shantanugadgil** [comment]
Thanks! Works now!
